### PR TITLE
remove submariner CRDs from csv bundle

### PR DESCRIPTION
**Pull request by zhiweiyin318**
relates issue: https://github.com/open-cluster-management/backlog/issues/8367

### CODE REVIEW

File: pkg/helpers/helpers.go [hunk 1]
```diff
@@ -125,6 +130,65 @@ func UpdateSubmarinerConfigStatusFn(cond metav1.Condition, managedClusterInfo co
 	}
 }
 
+type CRDClientHolder struct {
+	apiExtensionsClient apiextensionsclient.Interface
+}
+
+func NewCRDClientHolder() *CRDClientHolder {
```
Comment (qiujian16) [inline pkg/helpers/helpers.go:137]: why we just use clientholder in library go directly

File: pkg/hub/submarinerbroker/brokerCRDsController.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+		}
+	}
+
+	return operatorhelpers.NewMultiLineAggregate(errs)
+}
+
+func (c *submarinerBrokerCRDsController) cleanUp(ctx context.Context) error {
+	errs := []error{}
+	for _, crd := range brokerCRDNames {
+		err := c.crdClient.ApiextensionsV1().CustomResourceDefinitions().Delete(ctx, crd, metav1.DeleteOptions{})
```
Comment (qiujian16) [inline pkg/hub/submarinerbroker/brokerCRDsController.go:118]: do we get into this code? the crd is remove only when operator is deleted from olm.  Will olm guarantee the sequence that crd is delete at first, then operator?

File: pkg/helpers/helpers.go [hunk 3]
```diff
@@ -125,6 +130,65 @@ func UpdateSubmarinerConfigStatusFn(cond metav1.Condition, managedClusterInfo co
...
+	return &CRDClientHolder{}
+}
+func (c *CRDClientHolder) WithAPIExtensionsClient(client apiextensionsclient.Interface) *CRDClientHolder {
+	c.apiExtensionsClient = client
+	return c
+}
+
+// ApplyCRDDirectly is used to apply CRD v1 and v1beta1 since resourceapply in library-go cannot apply CRD v1 with error
+func ApplyCRDDirectly(
```
Comment (qiujian16) [inline pkg/helpers/helpers.go:146]: we can  live  with this, but we should submit a pr  to library-go.

File: pkg/hub/submarinerbroker/brokerCRDsController.go (hunk 2 above)
Comment (skeeey) [reply pkg/hub/submarinerbroker/brokerCRDsController.go:118]: not sure...

File: pkg/helpers/helpers.go (hunk 1 above)
Comment (zhiweiyin318) [reply pkg/helpers/helpers.go:137]: `apiExtensionsClient` in `clientholder` is an internal, cannot call outside.

File: pkg/hub/submarinerbroker/brokerCRDsController.go (hunk 2 above)
Comment (zhiweiyin318) [reply pkg/hub/submarinerbroker/brokerCRDsController.go:118]: I think CRD is the last one to be deleted in uninstall Operator process (refer https://github.com/operator-framework/olm-book/blob/master/docs/uninstall-an-operator.md)  
cleanup is not needed here.

**Pull request merged**